src/bin/travis/resources/jes_centaur.conf.ctmpl

```diff
@@ -0,0 +1,71 @@
+{{with $refreshToken := vault (printf "secret/dsde/cromwell/common/refresh-token")}}
```

Comment:
Does this file replace what's currently at `gs://cloud-cromwell-dev/travis-centaur/multiBackend.conf` ?